index.html

```diff
@@ -10,951 +12,1067 @@
                   </span>
                   <span>Examples</span>
-                </a>
+                </button>
               </span>
               <span class="navbar-item">
-                <a class="button is-white is-outlined">
+                <button class="button is-white is-outlined" aria-label="go to the documentation page">
                   <span class="icon">
-                    <i class="fa fa-book"></i>
+                    <i aria-hidden="true" class="fa fa-book" aria-hidden="true"></i>
                   </span>
                   <span>Documentation</span>
-                </a>
+                </button>
               </span>
               <span class="navbar-item">
-                <a class="button is-white is-outlined">
+                <button class="button is-white is-outlined" aria-label="go to the View Source page">
                   <span class="icon">
-                    <i class="fab fa-github"></i>
+                    <i aria-hidden="true" class="fab fa-github" aria-hidden="true"></i>
                   </span>
                   <span>View Source</span>
-                </a>
+                </button>
               </span>
-            </div>
+            </nav>
           </div>
         </div>
       </div>
     </div>
 
-    <div class="hero-body">
+    <section class="hero-body" aria-label="Subscribtion" role="banner">
       <div class="container has-text-centered">
         <div class="column is-6 is-offset-3">
-          <div class="title has-text-white">We have web accessibility in mind</div>
-          <div class="subtitle has-text-white">
+          <h2 class="title has-text-white">We have web accessibility in mind</h2>
+          <p id="subscribtion" class="subtitle has-text-white">
             Would you like to get the latest news?
-          </div>
-          <div class="box">
-            <div class="field is-grouped">
-              <p class="control is-expanded">
-                <input class="input">
-              </p>
-              <p class="control">
-                <a class="button is-info">Notify Me</a>
-              </p>
-            </div>
-          </div>
+					</p>
+          <form aria-labelledby="subscribtion" class="box no-border">
+            <fieldset class="field is-grouped">
+							<legend class="visuallyhidden">You can enter email address for email subscribtion and click "Notify Me" button.</legend>
+              <label for="emailInput" class="control is-expanded visuallyhidden">Enter your email</label>
+              <input aria-labelledby="subscribtion" name="emailInput" id="emailInput">
+              <button aria-labelledby="subscribtion" id="subscribeButton" name="subscribeButton" type="submit" class="button is-info">Notify Me</button>
+            </fieldset>
+          </form>
         </div>
       </div>
-    </div>
-  </div>
+    </section>
+  </header>
+	<main id="maincontent" role="main">
```

Comment:
role :)

Reply:
Thank you, I will remove the role in the next commit.